README.md

```diff
@@ -4,10 +4,16 @@ Plugin for Elasticsearch that allows users to filter locations using the Travelt
 ## Repository structure
 We maintain separate codebases for each ElasticSearch version between 7.10 and 8.1 (inclusive), and version 5.5. They reside in the appropriate folders (e.g. the plugin for version 7.16 is in the `7.16` folder).
 
-## Installation & configuration 
-This is a standard Elasticsearch plugin. As such it can be installed by running `elasticsearch-plugin install https://github.com/traveltime-dev/traveltime-elasticsearch-plugin/releases/download/${PLUGIN_VERSION}/traveltime-elasticsearch-plugin_${PLUGIN_VERSION}_${ES_VERSION}.zip`, where `PLUGIN_VERSION` is the latest plugin version available at https://github.com/traveltime-dev/traveltime-elasticsearch-plugin/releases (e.g. `v0.2.35`), and `ES_VERSION` is the exact version string of the Elasticsearch instance the plugin will be installed to (e.g. `7.17.14`)
+## Installation
+This is a standard Elasticsearch plugin. As such it can be installed by running `elasticsearch-plugin install https://github.com/traveltime-dev/traveltime-elasticsearch-plugin/releases/download/${PLUGIN_VERSION}/traveltime-elasticsearch-plugin_${PLUGIN_VERSION}_${ES_VERSION}.zip`,
+where `PLUGIN_VERSION` is the latest plugin version available at https://github.com/traveltime-dev/traveltime-elasticsearch-plugin/releases (e.g. `v0.2.35`),
+and `ES_VERSION` is the exact version string of the Elasticsearch instance the plugin will be installed to (e.g. `7.17.14`).
+For example, to install `traveltime-elasticsearch-plugin` you could run: `sudo bin/elasticsearch-plugin install https://github.com/traveltime-dev/traveltime-elasticsearch-plugin/releases/download/v0.2.37/traveltime-elasticsearch-plugin_v0.2.37_8.13.0.zip`
```

Comment:
Remove the example, I've found that users will just copy this command without checking if `v0.2.37` is indeed the latest plugin version.

Reply:
Hmm, maybe we could generate then the example. Or we would need to update it every time new version comes out.

Reply:
Generating on release would be nice, we could stick to manual updates only if we can have a CI action on release that makes sure the README is updated.